File: Java/ListaEstatica/ListaEstatica/src/Listavel.java
public interface Listavel <T> {
    // métodos 1 //
    void anexar(T dado);
    void inserir (int posicao, T dado);
    T selecionar(int posicao);
    T[] selecionarTodos();
    void atualizar(int posicao, T dado);
    T apagar(int posicao);
    void apagarTodos();

    // métodos 2 //
    boolean estaCheia();
    boolean estaVazia();
    String imprimir();
}

File: Kotlin/ListaEstatica/ListaEstatica/src/Listavel.kt
interface Listavel {
    //  métodos 1 //
    fun anexar(dado: Any?)
    fun inserir(posicao: Int, dado: Any?)
    fun selecionar(posicao: Int) : Any?
    fun selecionarTodos(): Array<Any?>
    fun atualizar(posicao: Int, dado: Any?)
    fun apagar(posicao: Int): Any?
    fun apagarTodos()

    // métodos 2 //
    fun estaCheia(): Boolean
    fun estaVazia(): Boolean
    fun imprimir(): String
}